### PR TITLE
Added Doc link to README.md

### PR DESCRIPTION
**Pull request by jalaja-git**
Added Doc link to README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
...
+To keep the code base and the API simple, zlog focuses on JSON logging only. 
+
+![](pretty.png)
+
+## Features
+
+* Blazing fast
+* Low to zero allocation
+* Level logging
+* Contextual fields
```
Comment (rchamarthy) [inline README.md]: * Context Based Logging
* Log Level support
* Near-Zero allocation

File: README.md [hunk 2]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
...
+
+The zlog package provides a fast and simple logger dedicated to JSON output.
+
+Zlog's API is designed to provide both a great developer experience and stunning [performance](#performance). Its unique chaining API allows zlog to write JSON log events by avoiding allocations and reflection.
+
+Refer complete documentation : https://godoc.org/github.com/atom-deps/zlog
+
+To keep the code base and the API simple, zlog focuses on JSON logging only. 
+
+![](pretty.png)
```
Comment (rchamarthy) [inline README.md]: Upload pretty.png

File: README.md [hunk 3]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
...
+* Level logging
+* Contextual fields
+
+## Usage
+
+```go
+import "github.com/atom-deps/zlog/log"
+```
+
+### A global logger can be use for simple logging
```
Comment (rchamarthy) [inline README.md]: No global logger EVER!

File: README.md [hunk 4]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
...
+
+```go
+log.Print("hello world")
```
Comment (rchamarthy) [inline README.md]: No log.Print, it must always be level and message.

File: README.md [hunk 5]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
...
+                 Str("component": "foo").
+                 Logger()
+sublogger.Info().Msg("hello world")
+
+// Output: {"level":"info","time":1494567715,"message":"hello world","component":"foo"}
+```
+
+
+### Log with no level nor message
+
```
Comment (rchamarthy) [inline README.md]: Not allowed and we wont add it.

File: README.md [hunk 6]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
...
+```go
+log.Log().Str("foo","bar").Msg("")
+
+// Output: {"time":1494567715,"foo":"bar"}
+```
+
+### Add contextual fields to the global logger
+
```
Comment (rchamarthy) [inline README.md]: No global logger.

File: README.md [hunk 7]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
...
+```go
+log.Logger = log.With().Str("foo", "bar").Logger()
+```
+
+
+### Pass a sub-logger by context
+
```
Comment (rchamarthy) [inline README.md]: Remove context.Context support for now. We will add it when we absolutely need it.

File: README.md [hunk 8]
```diff
@@ -6,3 +6,131 @@ zlog aims to provide a consistent, easy to use structured logging interface.
 
 The interface is inspired by [zerolog](https://github.com/rs/zerolog) and uses
 it for the default implementation.
+
+The zlog package provides a fast and simple logger dedicated to JSON output.
```
[the remark this answers is not in the document]
Comment (jalaja-git) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (jalaja-git) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (jalaja-git) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (jalaja-git) [reply README.md]: removed global

File: README.md (hunk 4 above)
Comment (jalaja-git) [reply README.md]: ok

File: README.md (hunk 5 above)
Comment (jalaja-git) [reply README.md]: ok

File: README.md (hunk 6 above)
Comment (jalaja-git) [reply README.md]: ok

**Pull request closed without merging**